client/src/pages/cv/index.tsx

```diff
@@ -0,0 +1,164 @@
+import * as React from 'react';
+import { Layout, Switch, Typography, Result } from 'antd';
+import { LoadingScreen } from 'components/LoadingScreen';
+import { NextRouter, withRouter } from 'next/router';
+import withSession, { Session } from 'components/withSession';
+import { Header, FooterLayout } from 'components';
+import FormCV from 'components/cv/EditCV';
+import ViewCV from 'components/cv/ViewCV';
+import NoConsentViewCV from 'components/cv/NoConsentViewCV';
+
+import { UserService } from '../../services/user';
+
+const { Content } = Layout;
+const { Text } = Typography;
+
+type Props = {
+  router: NextRouter;
+  session: Session;
+};
+
+type State = {
+  isLoading: boolean;
+  editMode: boolean;
+  opportunitiesConsent: boolean | null;
+  errorOccured: boolean;
+};
+
+class CVPage extends React.Component<Props, State> {
```

Comment:
same here. Please convert it to functional component. Let me know if you need any help.